nifi-extension-bundles/nifi-extension-utils/nifi-event-listen/src/main/java/org/apache/nifi/processor/util/listen/AbstractListenEventProcessor.java

```diff
@@ -56,13 +61,14 @@
  *
  * @param <E> the type of events being produced
  */
-public abstract class AbstractListenEventProcessor<E extends Event> extends AbstractProcessor {
+public abstract class AbstractListenEventProcessor<E extends Event> extends AbstractProcessor implements ListenComponent {
 
     public static final PropertyDescriptor PORT = new PropertyDescriptor
             .Builder().name("Port")
             .description("The port to listen on for communication.")
             .required(true)
             .expressionLanguageSupported(ExpressionLanguageScope.ENVIRONMENT)
+            .identifiesListenPort(TransportProtocol.UDP)
```

Comment:
```suggestion
            .identifiesListenPort(TransportProtocol.TCP)
```

Reply:
The only concrete classes extending `AbstractListenEventProcessor` are currently `ListenUDP` and `ListenUDPRecord`. That said, if you think this base class could be used for other transport protocols, I am happy to push the `PORT` property descriptor down into the extending classes rather than assuming the Abstract class is only intended for UDP.

Reply:
Oh good catch. I think that was a copy/paste error. Will update.